deltachat-ffi/deltachat.h

```diff
@@ -1343,6 +1343,24 @@ int             dc_estimate_deletion_cnt    (dc_context_t* context, int from_ser
 dc_array_t*     dc_get_fresh_msgs            (dc_context_t* context);
 
 
+/**
+ * Returns the message IDs of all messages of any chat
+ * with a database ID higher or equal than given.
```

Comment:
does the bot code save the last processed messageid then?
should we recommend to store that value in a `ui.` config key?